docs/gamejam-ref.json

```diff
@@ -1,3 +1,3 @@
 {
-    "appref": "/gamejam/traffic"
+    "appref": "gamejam/traffic"
```

Comment:
ahh this needs to be 

```
    "redirect": "/gamejam/traffic"
```

https://github.com/microsoft/pxt/blob/master/docs/index-ref.json listed under backend docs README if you search for `-ref`

Reply:
oh that's just a regular redirect though, it'll change the window location.... i guess that's just window dressing though

Reply:
i was hoping the appref thing would work generically but it looks like it is perhaps only for different versions of the editor